spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/TcpOutboundGatewayTests.java

```diff
@@ -1011,7 +1002,7 @@ public void postSend(Message<?> message, MessageChannel channel, boolean sent) {
 	}
 
 	@Test
-	public void testAsyncTimeout() throws Exception {
```

Comment:
Just for info: IDEA doesn't mark such a `public` modifier as redundant. 
So, we don't need to bother ourselves to remove them any more 😄 

(If it doesn't annoy you, of course)